src/openforms/js/components/formio_builder/translation.js

```diff
@@ -52,25 +52,45 @@ export const extractComponentLiterals = component => {
   ];
 
   const isDateOrDatetimeComponent = ['date', 'datetime'].includes(component.type);
-  const literals = allTranslatableProperties
+  const finalTranslatableProperties = allTranslatableProperties
     // Do not show warnings for date/datetime placeholders, they are not translatable:
     .filter(property => (isDateOrDatetimeComponent ? property !== 'placeholder' : property))
     .filter(property => !isRegex(property))
-    .map(property => get(component, property));
+    .map(property => {
+      return {
+        property: property,
+        literal: get(component, property),
+      };
+    });
```

Comment:
```suggestion
    .map(property => ({property, literal: get(component, property)}));
```

Could be also applied for the two below?

Reply:
I didn't do that there since it is specifically for the label.

Reply:
My suggestion does not introduce any changes, it just makes the syntax shorter

Reply:
aaah I thought you meant using the same approach but you mean only removing the `return` etc.